content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -4,198 +4,139 @@ Start receiving tickets
 
 Odoo *Helpdesk* offers multiple channels where customers can reach out for assistance, such as
 email, live chat, and through a website's submission form. The variety of these contact options
-provides customers with multiple opportunities to receive support quickly, while also giving the
-support team the ability to manage multi-channel support tickets from one central location.
+provides customers with multiple opportunities to receive support quickly while also allowing the
+support team to manage multi-channel support tickets from one central location.
 
 Enable channel options to submit tickets
 ========================================
 
-Go to :menuselection:`Helpdesk --> Configuration --> Teams` and choose an existing team, or click
-:guilabel:`New` to :doc:`create a new team <getting_started>`.
+Go to :menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams` and choose an existing
+team, or click :guilabel:`New` to :doc:`create a new team <getting_started>`.
 
-On the team's settings page, scroll down to :guilabel:`Channels`. Select one or more channel(s) to
-enable by checking the respective box(es).
+On the team's settings page, scroll down to the :guilabel:`Channels` and :guilabel:`Help Center`
+sections. Enable one or more channels by checking the respective boxes.
 
-- :guilabel:`Email Alias`
-- :guilabel:`Website Form`
-- :guilabel:`Live Chat`
+- :ref:`Email Alias <receiving_tickets/email-alias>`
+- :ref:`Live Chat <receiving_tickets/live-chat>`
+- :ref:`Website Form <receiving_tickets/website-form>`
```

Comment:
No, those are part of the help center, they are not considered channels. According to the PO, the webform was regrouped under the help center heading with 17 because it is required to enable any of the other Help Center functions (elearning, knowledge, or forums). However, it is still considered a channel. Help center is a different document entirely